source/js/buyers-guide/components/social-share/product-quiz-share-buttons.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import copyToClipboard from "../../../copy-to-clipboard";
+
+const tabletBreakpoint = 768;
+
+class ProductQuizShareButtons extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isMobileOrTablet: window.innerWidth < tabletBreakpoint,
+    };
+  }
+
+  componentDidMount() {
+    window.addEventListener("resize", this.handleResize);
+  }
+
+  handleResize() {
+    this.setState({
+      isMobileOrTablet: window.innerWidth < tabletBreakpoint,
+    });
+  }
+
+  shareButtonClicked(event, shareProgressButtonId) {
+    if (shareProgressButtonId) {
+      let shareProgressButton = document.querySelector(
+        `#${shareProgressButtonId} a`
+      );
+
+      if (shareProgressButton) {
+        shareProgressButton.click();
+      }
+    } else {
+      const linkButton = event.target;
+
+      linkButton.classList.add("copied");
+      linkButton.title = window.gettext("Copied");
+      const spanElements = linkButton.querySelectorAll("span");
+
+      // Iterate through each <span> element and update its inner text
```

Comment:
Since the copy button features two spans (one that is "Screenreader only" for mobile and one that renders the word "copy", we are updating both to "Copied", that way if a user clicks copy on mobile and resizes the screen, the desktop button will still say "Copied" as well